<!DOCTYPE html>
<html lang="en">

<head>
    <title>Parentage Analysis</title>
    <base href=".">
    <meta name="pathname" content="parentage-analysis.html">
    <meta name="description" content="Research - Parentage Analysis">
    <meta property="og:title" content="Parentage Analysis">
    <meta property="og:description" content="Research - Parentage Analysis">
    <meta property="og:type" content="website">
    <meta property="og:url" content="parentage-analysis.html">
    <meta property="og:image" content="undefined">
    <meta property="og:site_name" content="Research">
    <meta name="viewport"
        content="width=device-width, initial-scale=1.0, user-scalable=yes, minimum-scale=1.0, maximum-scale=5.0">
    <meta charset="UTF-8">
    <link rel="alternate" type="application/rss+xml" title="RSS Feed" href="site-lib/rss.xml">
    <script async="" id="webpage-script" src="site-lib/scripts/webpage.js"
        onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script>
    <script async="" id="graph-wasm-script" src="site-lib/scripts/graph-wasm.js"
        onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script>
    <script async="" id="graph-render-worker-script" src="site-lib/scripts/graph-render-worker.js"
        onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script>
    <link rel="icon" href="site-lib/media/favicon.png">
    <link rel="stylesheet" href="site-lib/styles/obsidian.css">
    <link rel="preload" href="site-lib/styles/other-plugins.css" as="style"
        onload="this.onload=null;this.rel='stylesheet'"><noscript>
        <link rel="stylesheet" href="site-lib/styles/other-plugins.css">
    </noscript>
    <link rel="stylesheet" href="site-lib/styles/theme.css">
    <link rel="preload" href="site-lib/styles/global-variable-styles.css" as="style"
        onload="this.onload=null;this.rel='stylesheet'"><noscript>
        <link rel="stylesheet" href="site-lib/styles/global-variable-styles.css">
    </noscript>
    <link rel="preload" href="site-lib/styles/supported-plugins.css" as="style"
        onload="this.onload=null;this.rel='stylesheet'"><noscript>
        <link rel="stylesheet" href="site-lib/styles/supported-plugins.css">
    </noscript>
    <link rel="preload" href="site-lib/styles/main-styles.css" as="style"
        onload="this.onload=null;this.rel='stylesheet'"><noscript>
        <link rel="stylesheet" href="site-lib/styles/main-styles.css">
    </noscript>
    <style>
        body {
            --line-width: 40em;
            --line-width-adaptive: 40em;
            --file-line-width: 40em;
            --sidebar-width: min(20em, 80vw);
            --collapse-arrow-size: 11px;
            --tree-vertical-spacing: 1.3em;
            --sidebar-margin: 12px
        }

        :root {
            background-color: #202124
        }

        .sidebar {
            height: 100%;
            font-size: 14px;
            z-index: 10;
            min-width: calc(var(--sidebar-width) + var(--divider-width-hover));
            max-width: calc(var(--sidebar-width) + var(--divider-width-hover));
            position: relative;
            overflow: hidden;
            overflow: clip;
            transition: min-width ease-in-out, max-width ease-in-out;
            transition-duration: .2s;
            contain: size
        }

        #left-sidebar {
            left: 0
        }

        #right-sidebar {
            right: 0
        }

        .sidebar.is-collapsed {
            min-width: 0;
            max-width: 0
        }

        .sidebar.floating {
            position: absolute
        }

        .sidebar .leaf-content {
            height: 100%;
            min-width: calc(var(--sidebar-width) - var(--divider-width-hover));
            top: 0;
            padding: var(--sidebar-margin);
            padding-top: 4em;
            line-height: var(--line-height-tight);
            background-color: var(--background-secondary);
            transition: background-color, border-right, border-left, box-shadow;
            transition-duration: var(--color-fade-speed);
            transition-timing-function: ease-in-out;
            position: absolute;
            display: flex;
            flex-direction: column
        }

        .sidebar:not(.is-collapsed) .leaf-content {
            min-width: calc(max(100%, var(--sidebar-width)) - 3px);
            max-width: calc(max(100%, var(--sidebar-width)) - 3px)
        }

        #left-sidebar-content {
            left: 0;
            border-top-right-radius: var(--radius-l);
            border-bottom-right-radius: var(--radius-l)
        }

        #right-sidebar-content {
            right: 0;
            border-top-left-radius: var(--radius-l);
            border-bottom-left-radius: var(--radius-l)
        }

        .sidebar #left-sidebar-content,
        .sidebar #right-sidebar-content {
            contain: none !important;
            container-type: normal !important;
            animation: none !important
        }

        .sidebar:has(.leaf-content:empty):has(.topbar-content:empty) {
            display: none
        }

        .sidebar-topbar {
            height: 2em;
            width: var(--sidebar-width);
            top: var(--sidebar-margin);
            padding-inline: var(--sidebar-margin);
            z-index: 1;
            position: fixed;
            display: flex;
            align-items: center;
            transition: width ease-in-out;
            transition-duration: inherit
        }

        .sidebar.is-collapsed .sidebar-topbar {
            width: calc(2.3em + var(--sidebar-margin) * 2)
        }

        .sidebar .sidebar-topbar.is-collapsed {
            width: 0
        }

        #left-sidebar .sidebar-topbar {
            left: 0;
            flex-direction: row
        }

        #right-sidebar .sidebar-topbar {
            right: 0;
            flex-direction: row-reverse
        }

        #left-sidebar .topbar-content {
            margin-right: calc(2.3em + var(--sidebar-margin));
            flex-direction: row
        }

        #right-sidebar .topbar-content {
            margin-left: calc(2.3em + var(--sidebar-margin));
            flex-direction: row-reverse
        }

        .topbar-content {
            overflow: hidden visible;
            overflow: clip visible;
            width: 100%;
            height: 100%;
            display: flex;
            align-items: center;
            transition: inherit
        }

        .sidebar.is-collapsed .topbar-content {
            width: 0;
            transition: inherit
        }

        .clickable-icon.sidebar-collapse-icon {
            background-color: transparent;
            color: var(--icon-color-focused);
            padding: 2px !important;
            margin: 0 !important;
            height: 100% !important;
            width: 2.3em !important;
            margin-inline: 0.14em !important;
            position: absolute
        }

        #left-sidebar .clickable-icon.sidebar-collapse-icon {
            transform: rotateY(180deg);
            right: var(--sidebar-margin)
        }

        #right-sidebar .clickable-icon.sidebar-collapse-icon {
            transform: rotateY(180deg);
            left: var(--sidebar-margin)
        }

        .clickable-icon.sidebar-collapse-icon svg.svg-icon {
            width: 100%;
            height: 100%
        }

        .feature-title {
            margin-left: 4px;
            text-transform: uppercase;
            letter-spacing: .06em;
            margin-top: .75em;
            margin-bottom: .75em
        }

        .feature-header {
            display: flex;
            align-items: center;
            padding-top: 0;
            font-size: 1em;
            padding-left: 0
        }

        body.floating-sidebars .sidebar {
            position: absolute
        }

        body {
            transition: background-color var(--color-fade-speed) ease-in-out
        }

        #layout {
            display: flex;
            flex-direction: row;
            height: 100%;
            width: 100%;
            align-items: stretch;
            justify-content: center
        }

        #center-content {
            flex-basis: 100%;
            max-width: 100%;
            width: 100%;
            height: 100%;
            display: flex;
            flex-direction: column;
            align-items: center;
            transition: opacity .2s ease-in-out;
            contain: inline-size
        }

        .hide {
            opacity: 0 !important;
            transition: opacity .2s ease-in-out;
            pointer-events: none
        }

        #center-content>.obsidian-document {
            padding-left: 2em;
            padding-right: 1em;
            margin-bottom: 0;
            width: 100%;
            width: -webkit-fill-available;
            width: -moz-available;
            width: fill-available;
            transition: background-color var(--color-fade-speed) ease-in-out;
            border-top-right-radius: var(--window-radius, var(--radius-m));
            border-top-left-radius: var(--window-radius, var(--radius-m));
            overflow-x: hidden !important;
            overflow-y: auto !important;
            display: flex !important;
            flex-direction: column !important;
            align-items: center !important;
            contain: inline-size
        }

        body #center-content>.obsidian-document>.markdown-preview-sizer {
            padding-bottom: 80vh;
            width: 100%;
            max-width: var(--line-width);
            flex-basis: var(--line-width);
            transition: background-color var(--color-fade-speed) ease-in-out;
            contain: inline-size
        }

        #center-content>.obsidian-document>div {
            width: 100% !important;
            transition: background-color var(--color-fade-speed) ease-in-out;
            contain: inline-size
        }

        #center-content>.obsidian-document:not([data-type=markdown]).embed {
            display: flex;
            padding: 1em;
            height: 100%;
            width: 100%;
            align-items: center;
            justify-content: center
        }

        #center-content>.obsidian-document:not([data-type=markdown]).embed>* {
            max-width: 100%;
            max-height: 100%;
            object-fit: contain
        }

        :not(h1, h2, h3, h4, h5, h6):has(> :is(.math:not(.math-inline), table)) {
            overflow-x: auto !important
        }

        #center-content>.obsidian-document:not([data-type=markdown]) {
            overflow-x: auto;
            contain: content;
            padding: 0;
            margin: 0;
            height: 100%
        }

        .obsidian-document[data-type=attachment] {
            display: flex;
            flex-direction: column;
            align-items: center;
            justify-content: center;
            height: 100%;
            width: 100%
        }

        .obsidian-document[data-type=attachment]>* {
            outline: 0;
            border: none;
            box-shadow: none
        }

        .obsidian-document[data-type=attachment] :is(img) {
            max-width: 90%;
            max-height: 90%;
            object-fit: contain
        }

        .obsidian-document[data-type=attachment]>:is(audio) {
            width: 100%;
            max-width: min(90%, var(--line-width))
        }

        .obsidian-document[data-type=attachment]>:is(embed, iframe, video) {
            width: 100%;
            height: 100%;
            max-width: 100%;
            max-height: 100%;
            object-fit: contain
        }

        .canvas-wrapper>:is(.header, .footer) {
            z-index: 100;
            position: absolute;
            display: flex;
            justify-content: center;
            flex-direction: column;
            width: 100%;
            align-items: center
        }

        .scroll-highlight {
            position: absolute;
            width: 100%;
            height: 100%;
            pointer-events: none;
            z-index: 1000;
            background-color: hsla(var(--color-accent-hsl), .25);
            opacity: 0;
            padding: 1em;
            inset: 50%;
            translate: -50% -50%;
            border-radius: var(--radius-s)
        }
    </style>
    <script
        defer="">async function loadIncludes() { let e = document.querySelectorAll("link[itemprop='include']"); for (const t of e) { let e = t.getAttribute("href"); try { let o = ""; if (e.startsWith("https:") || e.startsWith("http:") || "file:" != window.location.protocol) { const l = await fetch(e); if (!l.ok) { console.log("Could not include file: " + e), t?.remove(); continue } o = await l.text() } else { const t = document.getElementById(btoa(encodeURI(e))); if (t) { const e = JSON.parse(decodeURI(atob(t.getAttribute("value") ?? ""))); o = e?.data ?? "" } } let l = document.createRange().createContextualFragment(o), n = Array.from(l.children); for (let e of n) e?.classList?.add("hide"), e.style.transition = "opacity 0.5s ease-in-out", setTimeout((() => { e?.classList?.remove("hide") }), 10); t.before(l), t.remove(), console.log("Included text: " + o), console.log("Included file: " + e) } catch (o) { t?.remove(), console.log("Could not include file: " + e, o); continue } } } document.addEventListener("DOMContentLoaded", (() => { loadIncludes() })); let isFileProtocol = "file:" == location.protocol; function waitLoadScripts(e, t) { let o = e.map((e => document.getElementById(e + "-script"))); !function e(l) { let n = o[l], c = l + 1; n ? (n && "true" != n.getAttribute("loaded") || l < o.length && e(c), l < o.length && n.addEventListener("load", (() => e(c)))) : l < o.length ? e(c) : t() }(0) }</script>
</head>

<body
    class="publish css-settings-manager mod-windows is-hidden-frameless show-inline-title show-ribbon show-view-header minimal-default-light minimal-nord-dark minimal-readable-off colorful-headings colorful-active links-int-on links-ext-on full-width-media is-focused">
    <script
        defer="">let theme = localStorage.getItem("theme") || (window.matchMedia("(prefers-color-scheme: dark)").matches ? "dark" : "light"); "dark" == theme ? (document.body.classList.add("theme-dark"), document.body.classList.remove("theme-light")) : (document.body.classList.add("theme-light"), document.body.classList.remove("theme-dark")), window.innerWidth < 480 ? document.body.classList.add("is-phone") : window.innerWidth < 768 ? document.body.classList.add("is-tablet") : window.innerWidth < 1024 ? document.body.classList.add("is-small-screen") : document.body.classList.add("is-large-screen")</script>
    <div class="parsed-feature-container" style="display: contents;">
        <link itemprop="include" href="site-lib/html/custom-head-content-content.html">
    </div>
    <div id="layout">
        <div id="left-content" class="leaf" style="--sidebar-width: var(--sidebar-width-left);"></div>
        <div id="center-content" class="leaf">
            <div class="obsidian-document markdown-preview-view markdown-rendered node-insert-event allow-fold-headings allow-fold-lists show-indentation-guide show-properties is-readable-line-width"
                data-type="markdown">
                <style id="MJX-CHTML-styles"></style>
                <div class="markdown-preview-sizer markdown-preview-section">
                    <div class="header">
                        <h1 class="page-title heading inline-title" id="_0"></h1>
                        <div class="data-bar"></div>
                    </div>
                    <div class="markdown-preview-pusher" style="width: 1px; height: 0.1px; margin-bottom: 0px;"></div>

                    <div class="el-h2">
                        <h2 data-heading="Email responses" dir="auto" class="heading" id="Email_responses_0">
                            <span class="heading-collapse-indicator collapse-indicator collapse-icon"><svg
                                    xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"
                                    fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                                    stroke-linejoin="round" class="svg-icon right-triangle">
                                    <path d="M3 8L12 17L21 8"></path>
                                </svg></span>
                            Email responses
                        </h2>
                    </div>
                    <div class="el-pre">
                        <!-- This container holds the tab navigation and content -->
                        <div class="block-language-tabs tab-panel-container highlight-selected-tab-name">
                            <!-- Tab navigation buttons -->
                            <div class="tab-scroll-container">
                                <ul class="tab-container">
                                    <li class="tab selected">
                                        <p dir="auto">Wildlife Genetics (Canada)</p>
                                    </li>
                                    <li class="tab">
                                        <p dir="auto">Oregon State- Epps Lab</p>
                                    </li>
                                    <li class="tab">
                                        <p dir="auto">East Stroudsburg University</p>
                                    </li>
                                    <li class="tab">
                                        <p dir="auto">Arizona Genetics Core</p>
                                    </li>
                                </ul>
                            </div>
                            <!-- Tab content panels -->
                            <div class="content-container">
                                <!-- Content for the first tab (Wildlife Genetics) - initially selected -->
                                <div class="selected">
                                    <div class="rendered-content">
                                        <p dir="auto">Contact: David Paetkau, <a data-tooltip-position="top"
                                                aria-label="mailto:dpaetkau@wildlifegenetics.ca" rel="noopener nofollow"
                                                class="external-link" href="mailto:dpaetkau@wildlifegenetics.ca"
                                                target="_blank">dpaetkau@wildlifegenetics.ca</a><br>
                                            <br>
                                            Hi Shannon,<br>
                                            <br>
                                            This is a genus that we haven't worked on before, so there is a bit to think
                                            about in terms of getting markers going. Normally we'd charge about US $78
                                            per sample for DNA extraction and 18-locus genotyping, plus about $1000 per
                                            project for error-checking and reporting. But for the first go with a
                                            species there will extra costs associated with primer design and
                                            testing.<br>
                                            <br>
                                            For a series of practical and technical reasons, we prefer to work with
                                            dinucleotide repeats over the tetras described in the Faircloth paper (I
                                            wasn't able to access the Fowler paper), and the Faircloth markers at least
                                            didn't show great variability - perhaps a characteristic of the study
                                            population, but perhaps to do with the loci themselves. Given the
                                            availability of a genome, my preference would be to search each chromosome
                                            for a tidy, long dinucleotide repeat locus, and design primers for about 50%
                                            more loci than we'd be aiming to use in the end; e.g. 27 loci if aiming for
                                            18. We charge $300 per locus to design and test primers, so that puts us
                                            about $8000 down the road before we find out if we ended up with nice
                                            markers. We would use about 30 samples for marker testing, so there would be
                                            no subsequent need to genotype those 30 samples, but it does mean that the
                                            first round of work would be around $5500 more expensive than a routine
                                            project on a species that we know.<br>
                                            <br>
                                            In short, if it's a project that might extend for a number of years, and run
                                            to many hundreds of samples, the development cost isn't too out of line
                                            (we've done similar things for a multiyear gyrfalcon project that I'm quite
                                            pleased with), but for a one-off project it might strike you as too
                                            expensive.<br>
                                            <br>
                                            With all the primer design and testing we'd need a solid 8 months to do a
                                            project like this.<br>
                                            <br>
                                            Thoughts?<br>
                                            <br>
                                            David
                                        </p>
                                    </div>
                                </div>
                                <!-- Content for the second tab (Oregon State) -->
                                <div>
                                    <div class="rendered-content">
                                        <p dir="auto">Contact: Rachel Crowhurst, <a data-tooltip-position="top"
                                                aria-label="mailto:Rachel.Crowhurst@oregonstate.edu"
                                                rel="noopener nofollow" class="external-link"
                                                href="mailto:Rachel.Crowhurst@oregonstate.edu"
                                                target="_blank">Rachel.Crowhurst@oregonstate.edu</a><br>
                                            <br>
                                            Hi Shannon,<br>
                                            <br>
                                            Thanks for your interest in our lab, and apologies for the delay in
                                            responding to you.<br>
                                            <br>
                                            I looked through the first paper that you sent (Faircloth et al. 2006) and
                                            there are plenty of highly variable markers for bluebirds, which is
                                            great.&nbsp; However, it looks like in that paper they ran every marker
                                            individually because they were characterizing new loci.&nbsp; For us to try
                                            to estimate how much this would cost we’d need to see how many markers can
                                            be multiplexed in a single PCR reaction, as that can greatly influence the
                                            cost.&nbsp; I could try to guess based on allele sizes, but it would be most
                                            helpful to see how someone else used these markers in a previously published
                                            study.&nbsp; Unfortunately, I was unable to access the Fowler Neal thesis
                                            without a University of Arkansas login.&nbsp; Would you be able to download
                                            the thesis and send it to me as a .pdf?&nbsp; &nbsp;We typically charge
                                            about $100 per sample for species that we’ve worked on before, for up to 3
                                            panels of markers.&nbsp; However, there is an initial cost for the labelled
                                            primers and since we do not have those on-hand, that would add approximately
                                            $4000.&nbsp; We would probably also need to build in a buffer for
                                            optimization runs, since it often takes a bit of tweaking before we can get
                                            a new primer/species system to work, even if we’re working off of a
                                            previously published study.&nbsp;<br>
                                            <br>
                                            Full disclosure – we specialize in mammalian samples (feces, blood, tissue)
                                            and although I’ve done some sexing work on blood from seabirds I do not have
                                            experience working with feathers.&nbsp; In my reading I’ve seen published
                                            protocols wherein people use the same DNA extraction kit for feathers as we
                                            use for muscle (Qiagen DNEasy blood and tissue), with a few
                                            modifications.&nbsp; However the Faircloth paper used DNA from blood and the
                                            PCR product sizes are very large (mostly 250-350bp) and it’s unclear to me
                                            whether the feather DNA would work as well for such large amplicons.&nbsp;
                                            I’d like to see what methods and sample type were used in Jessica Fowler
                                            Neal’s study.&nbsp; Once we have a better idea of how many markers we could
                                            fit into a single PCR reaction and what extraction methods would be best to
                                            try, we would be better positioned to estimate the cost and turnaround
                                            time.<br>
                                            <br>
                                            Best,<br>
                                            Rachel Crowhurst
                                        </p>
                                    </div>
                                </div>
                                <!-- Content for the third tab (East Stroudsburg) -->
                                <div>
                                    <div class="rendered-content">
                                        <p dir="auto">Contact: Nicole Chinnici, <a data-tooltip-position="top"
                                                aria-label="mailto:nchinnici@esu.edu" rel="noopener nofollow"
                                                class="external-link" href="mailto:nchinnici@esu.edu"
                                                target="_blank">nchinnici@esu.edu</a><br>
                                            <br>
                                            Hi Shannon<br>
                                            <br>
                                            Thank you for your request. This is a service that we can assist&nbsp;you
                                            with. To better understand your project and get you an accurate quote, we
                                            need some additional&nbsp;information.
                                        </p>
                                        <ol>
                                            <li dir="auto">Will you be selecting the microsatellites for your project?
                                            </li>
                                            <li dir="auto">Will you purchase those microsatellites yourself, or would
                                                you like that included in our service?</li>
                                            <li dir="auto">What is the timeframe you are looking to complete this
                                                project?</li>
                                        </ol>
                                        <p dir="auto"><br>
                                            Thank you,<br>
                                            Nicole</p>
                                    </div>
                                </div>
                                <!-- Content for the fourth tab (Arizona Genetics) -->
                                <div>
                                    <div class="rendered-content">
                                        <p dir="auto">Contact: Barbara Fransway, <a data-tooltip-position="top"
                                                aria-label="mailto:bbf@arizona.edu" rel="noopener nofollow"
                                                class="external-link" href="mailto:bbf@arizona.edu"
                                                target="_blank">bbf@arizona.edu</a><br>
                                            <br>
                                            Shannon- thank you for getting back to our lab.<br>
                                            <br>
                                            I don’t have experience extracting DNA from feathers, this would have to be
                                            piloted with some samples and then we could discuss how to proceed after DNA
                                            QC and yield assessment.<br>
                                            <br>
                                            For the updated costs-<br>
                                            <br>
                                            Assume- 5 multiplexes per sample- &nbsp;this will help keep the primer costs
                                            low and reduce the optimization effort. In addition, if there are markers
                                            that are “must have” vs “not critical” this will also be informative.<br>
                                            <br>
                                            In short, rough costs are:<br>
                                            <br>
                                            Once optimized-<br>
                                            DNA extraction +QC ~$12.50/sample<br>
                                            PCR $3.60/reaction (<em>5= $18)<br>
                                                Fragment analysis $2.10/well (</em>5=$10.50)<br>
                                            <br>
                                            So production test costs would be about $40/sample- this is Extraction, QC,
                                            PCR and Frag if we do 5 reactions per sample<br>
                                            <br>
                                            Primer costs are ~$85-$100/locus *18= $1550-1800<br>
                                            <br>
                                            Optimization: PCR- $3.60/reaction, gel electrophoresis estimated $60-90, and
                                            fragment analysis $2.10/sample- the quantities of PCR and Frag depend on how
                                            complex the multiplex assays are. Assume $500-600 for optimization<br>
                                            <br>
                                            Estimated total-<br>
                                            Pilot extraction/QC &nbsp;(n=8) $100 if successful then we proceed.<br>
                                            DNA extraction/QC, PCR and FRAG (n=100) $4000<br>
                                            Data Finishing $420<br>
                                            Optimization $600<br>
                                            Primers $1800<br>
                                            <br>
                                            So it looks like we are at about $6900 for this project. This again greatly
                                            depends on the loci selected and how we multiplex them together. In addition
                                            the pilot DNA extraction and following QC will be a critical point prior to
                                            proceeding with the samples.<br>
                                            <br>
                                            If you would like a more official quote please register for iLab and I can
                                            create one in this system for you. See registration instructions
                                            attached.<br>
                                            <br>
                                            Thank you and I look forward to being in contact soon.<br>
                                            Barbara
                                        </p>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                    <!-- The rest of your content follows -->
                    <div class="el-h2">...</div>

                    <div class="el-h2">
                        <h2 data-heading="Microsatellite Selection" dir="auto" class="heading"
                            id="Microsatellite_Selection_0"><span
                                class="heading-collapse-indicator collapse-indicator collapse-icon"><svg
                                    xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"
                                    fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                                    stroke-linejoin="round" class="svg-icon right-triangle">
                                    <path d="M3 8L12 17L21 8"></path>
                                </svg></span>Microsatellite Selection</h2>
                    </div>
                    <div class="el-p">
                        <p dir="auto"><strong>Characteristics for primer sets should be highly polymorphic (many
                                alleles) to distinguish between closely related individuals, and should maintain HWE for
                                reliable allele frequency data (i.e. H<sub>o</sub> ≈ H<sub>E</sub>)</strong></p>
                    </div>
                    <div class="el-h3">
                        <h3 data-heading="main texts" dir="auto" class="heading" id="main_texts_0"><span
                                class="heading-collapse-indicator collapse-indicator collapse-icon"><svg
                                    xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"
                                    fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                                    stroke-linejoin="round" class="svg-icon right-triangle">
                                    <path d="M3 8L12 17L21 8"></path>
                                </svg></span>main texts</h3>
                    </div>
                    <div class="el-ul">
                        <ul class="has-list-bullet">
                            <li data-line="0" dir="auto"><span class="list-bullet"></span><span
                                    class="list-collapse-indicator collapse-indicator collapse-icon"><svg
                                        xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"
                                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                                        stroke-linejoin="round" class="svg-icon right-triangle">
                                        <path d="M3 8L12 17L21 8"></path>
                                    </svg></span>
                                <p><strong>Gutiérrez et al. 2022:</strong> <a rel="noopener nofollow"
                                        class="external-link is-unresolved"
                                        href="https://www.biorxiv.org/content/10.1101/2022.05.26.493631v1.full.pdf"
                                        target="_self">https://www.biorxiv.org/content/10.1101/2022.05.26.493631v1.full.pdf</a>
                                </p>
                                <ul class="has-list-bullet">
                                    <li data-line="1" dir="auto"><span class="list-bullet"></span>"Five fluorescently
                                        labeled primers were amplified: Smex9 (Ferree et al. 2008), Smex14 (Ferree et
                                        al. 2008), Pdo5 (Griffith et al. 1999), Sialia36 (Faircloth et al. 2006), and
                                        Sialia37 (Faircloth et al. 2006)... Each primer was selected based on previous
                                        amplification success using eastern bluebird DNA (Faircloth et al. 2006, Ferree
                                        et al. 2008, and Stewart et al. 2009)." </li>
                                </ul>
                            </li>
                            <li data-line="3" dir="auto"><span class="list-bullet"></span><span
                                    class="list-collapse-indicator collapse-indicator collapse-icon"><svg
                                        xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"
                                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                                        stroke-linejoin="round" class="svg-icon right-triangle">
                                        <path d="M3 8L12 17L21 8"></path>
                                    </svg></span>
                                <p><strong>Stewart et al. 2009</strong>: <a rel="noopener nofollow"
                                        class="external-link is-unresolved"
                                        href="https://doi.org/10.1007/s00265-009-0862-y"
                                        target="_self">https://doi.org/10.1007/s00265-009-0862-y</a></p>
                                <ul class="has-list-bullet">
                                    <li data-line="4" dir="auto"><span class="list-bullet"></span>Uses Pdo5 (Griffith et
                                        al. 1999), a loci for paternity in HOSP, as well as other loci developed for
                                        other species. Only EABL specific loci from Balenger et al 2009.</li>
                                </ul>
                            </li>
                            <li data-line="6" dir="auto"><span class="list-bullet"></span><span
                                    class="list-collapse-indicator collapse-indicator collapse-icon"><svg
                                        xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"
                                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                                        stroke-linejoin="round" class="svg-icon right-triangle">
                                        <path d="M3 8L12 17L21 8"></path>
                                    </svg></span>
                                <p><strong>Balenger et al. 2009:</strong> <a rel="noopener nofollow"
                                        class="external-link is-unresolved"
                                        href="https://doi.org/10.1111/j.1600-048X.2009.04521.x"
                                        target="_self">https://doi.org/10.1111/j.1600-048X.2009.04521.x</a></p>
                                <ul class="has-list-bullet">
                                    <li data-line="7" dir="auto"><span class="list-bullet"></span>uses Sailia37 from
                                        Faircloth 2006, and 2 new primers (Eabl007, Eabl129).</li>
                                </ul>
                            </li>
                            <li data-line="9" dir="auto"><span class="list-bullet"></span><span
                                    class="list-collapse-indicator collapse-indicator collapse-icon"><svg
                                        xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"
                                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                                        stroke-linejoin="round" class="svg-icon right-triangle">
                                        <path d="M3 8L12 17L21 8"></path>
                                    </svg></span>
                                <p><strong>Ferree et al 2008</strong>: <a rel="noopener nofollow"
                                        class="external-link is-unresolved"
                                        href="https://doi.org/10.1111/j.1755-0998.2008.02265.x"
                                        target="_self">https://doi.org/10.1111/j.1755-0998.2008.02265.x</a></p>
                                <ul class="has-list-bullet">
                                    <li data-line="10" dir="auto"><span class="list-bullet"></span>Though primers
                                        labelled as Smex, tested on NC population of S. sialis as well</li>
                                    <li data-line="11" dir="auto"><span class="list-bullet"></span>The two primers
                                        chosen by Gutierrez 2022 had a significant deviation from Hardy–Weinberg
                                        equilibrium in this original paper--- my understanding is this is not
                                        desirable?? Possibly did some trouble shooting for their population to come to
                                        this selection?</li>
                                </ul>
                            </li>
                            <li data-line="13" dir="auto"><span class="list-bullet"></span>
                                <p><strong>Faircloth et al. 2006</strong>: <a rel="noopener nofollow"
                                        class="external-link is-unresolved"
                                        href="https://doi.org/10.1111/j.1471-8286.2006.01408.x"
                                        target="_self">https://doi.org/10.1111/j.1471-8286.2006.01408.x</a><br>
                                    -<u> Neal 2014</u>: chose Sialia 6, 15, 20, 23, 27, 28, 34, 36, 38 "based on
                                    successful amplification and a satisfactory number of alleles."</p>
                            </li>
                        </ul>
                    </div>
                    <div class="el-h3">
                        <h3 data-heading="additional texts" dir="auto" class="heading" id="additional_texts_0"><span
                                class="heading-collapse-indicator collapse-indicator collapse-icon"><svg
                                    xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"
                                    fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                                    stroke-linejoin="round" class="svg-icon right-triangle">
                                    <path d="M3 8L12 17L21 8"></path>
                                </svg></span>additional texts</h3>
                    </div>
                    <div class="el-ul">
                        <ul class="has-list-bullet">
                            <li data-line="0" dir="auto"><span class="list-bullet"></span>
                                <p><strong>Joly 2024:</strong> Mountain Bluebirds study, Chose Sailia 37, EABL 129, and
                                    Smex 6,8,10,13</p>
                            </li>
                            <li data-line="2" dir="auto"><span class="list-bullet"></span><span
                                    class="list-collapse-indicator collapse-indicator collapse-icon"><svg
                                        xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"
                                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                                        stroke-linejoin="round" class="svg-icon right-triangle">
                                        <path d="M3 8L12 17L21 8"></path>
                                    </svg></span>
                                <p><strong>Dickinson et al. 2016 &amp; Ferree et al. 2010:</strong> selected 11
                                    microsatellites from Ferree 2008 paper- Smex 2,4,5, 7-14</p>
                                <ul class="has-list-bullet">
                                    <li data-line="3" dir="auto"><span class="list-bullet"></span>In other words, they
                                        selected primers that did not deviate from HWE for western bluebirds</li>
                                    <li data-line="4" dir="auto"><span class="list-bullet"></span><span
                                            class="list-collapse-indicator collapse-indicator collapse-icon"><svg
                                                xmlns="http://www.w3.org/2000/svg" width="24" height="24"
                                                viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2"
                                                stroke-linecap="round" stroke-linejoin="round"
                                                class="svg-icon right-triangle">
                                                <path d="M3 8L12 17L21 8"></path>
                                            </svg></span>For EABL, this would be Smex 2-8,10,12,13 <em>(15 borderline
                                            HWE deviation)</em>
                                        <ul class="has-list-bullet">
                                            <li data-line="5" dir="auto"><span class="list-bullet"></span>whitling it
                                                down more -3, 5, 6, 7, 12, 13 are best for HWE and allele no.</li>
                                        </ul>
                                    </li>
                                </ul>
                            </li>
                            <li data-line="7" dir="auto"><span class="list-bullet"></span>
                                <p><strong>Avery et al. 2013:</strong> selected 12 loci from Faircloth 2006 (those
                                    without sig. HWE deviation and with adequate # of alleles)</p>
                            </li>
                            <li data-line="9" dir="auto"><span class="list-bullet"></span>
                                <p><strong>Liu 2009:</strong> Chose Sailia 11,22,27,36,37, eabl129, and MOBL47,49,53,86b
                                    from Balenger et al. 2009 (likely an early copy? Only 49 &amp; 87 published). MOBL53
                                    &amp; Sailia22 had HWE deviation.</p>
                            </li>
                        </ul>
                    </div>
                    <div class="el-blockquote">
                        <blockquote dir="auto">
                            <p>Based on HWE and allele #, it seems like these are good candidates?<br>
                                Faircloth- Sailia 11, 22, 28, 37 (trinucleotide)<br>
                                Balenger - Eabl007, Eabl129, (Maybe MOBL49, MOBL87?) (dinucleotide)<br>
                                Ferree - Smex 3,5,6,7,12,13 (dinucleotide)</p>
                        </blockquote>
                    </div>
                    <div class="el-h2">
                        <h2 data-heading="Other relevant info" dir="auto" class="heading" id="Other_relevant_info_0">
                            <span class="heading-collapse-indicator collapse-indicator collapse-icon"><svg
                                    xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"
                                    fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                                    stroke-linejoin="round" class="svg-icon right-triangle">
                                    <path d="M3 8L12 17L21 8"></path>
                                </svg></span>Other relevant info</h2>
                    </div>
                    <div class="el-p">
                        <p dir="auto">Eastern Bluebird whole genome assembly (LSU):</p>
                    </div>
                    <div class="el-ul">
                        <ul class="has-list-bullet">
                            <li data-line="0" dir="auto"><span class="list-bullet"></span><a rel="noopener nofollow"
                                    class="external-link is-unresolved"
                                    href="https://www.ncbi.nlm.nih.gov/datasets/genome/GCA_009812075.1/"
                                    target="_self">https://www.ncbi.nlm.nih.gov/datasets/genome/GCA_009812075.1/</a><br>
                                WGS on 6 individuals from Schroeder et al. 2024 (Rolland Lab):</li>
                            <li data-line="2" dir="auto"><span class="list-bullet"></span><a rel="noopener nofollow"
                                    class="external-link is-unresolved" href="https://doi.org/10.1002/ece3.10851"
                                    target="_self">https://doi.org/10.1002/ece3.10851</a></li>
                            <li data-line="3" dir="auto"><span class="list-bullet"></span><a rel="noopener nofollow"
                                    class="external-link is-unresolved"
                                    href="https://www.ncbi.nlm.nih.gov/bioproject/1040132"
                                    target="_self">https://www.ncbi.nlm.nih.gov/bioproject/1040132</a><br>
                                Genbank microsatellite sequences:</li>
                            <li data-line="5" dir="auto"><span class="list-bullet"></span><a rel="noopener nofollow"
                                    class="external-link is-unresolved"
                                    href="https://www.ncbi.nlm.nih.gov/nuccore/?term=sialia+microsatellite"
                                    target="_self">https://www.ncbi.nlm.nih.gov/nuccore/?term=sialia+microsatellite</a>
                            </li>
                        </ul>
                    </div>
                    <div class="footer">
                        <div class="data-bar"></div>
                    </div>
                </div>
            </div>
        </div>
        <div id="right-content" class="leaf" style="--sidebar-width: var(--sidebar-width-right);"></div>
    </div>
</body>

</html>