<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.56">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Parentage Analysis</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="ParentageAnalysis_files/libs/clipboard/clipboard.min.js"></script>
<script src="ParentageAnalysis_files/libs/quarto-html/quarto.js"></script>
<script src="ParentageAnalysis_files/libs/quarto-html/popper.min.js"></script>
<script src="ParentageAnalysis_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="ParentageAnalysis_files/libs/quarto-html/anchor.min.js"></script>
<link href="ParentageAnalysis_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="ParentageAnalysis_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="ParentageAnalysis_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="ParentageAnalysis_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="ParentageAnalysis_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">


</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Parentage Analysis</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<section id="email-responses" class="level3">
<h3 class="anchored" data-anchor-id="email-responses">Email responses</h3>
<div class="tabset-margin-container"></div><div class="panel-tabset" style="font-size:14px">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-1-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-1" role="tab" aria-controls="tabset-1-1" aria-selected="true" href="">Wildlife Genetics (Canada)</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-2" role="tab" aria-controls="tabset-1-2" aria-selected="false" href="">Oregon State- Epps Lab</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-3" role="tab" aria-controls="tabset-1-3" aria-selected="false" href="">East Stroudsburg University</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-4-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-4" role="tab" aria-controls="tabset-1-4" aria-selected="false" href="">Arizona Genetics Core</a></li></ul>
<div class="tab-content" style="font-size:14px">
<div id="tabset-1-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-1-1-tab">
<p>Contact: David Paetkau, dpaetkau@wildlifegenetics.ca</p>
<p>Hi Shannon,</p>
<p>This is a genus that we haven’t worked on before, so there is a bit to think about in terms of getting markers going. Normally we’d charge about US $78 per sample for DNA extraction and 18-locus genotyping, plus about $1000 per project for error-checking and reporting. But for the first go with a species there will extra costs associated with primer design and testing.</p>
<p>For a series of practical and technical reasons, we prefer to work with dinucleotide repeats over the tetras described in the Faircloth paper (I wasn’t able to access the Fowler paper), and the Faircloth markers at least didn’t show great variability - perhaps a characteristic of the study population, but perhaps to do with the loci themselves. Given the availability of a genome, my preference would be to search each chromosome for a tidy, long dinucleotide repeat locus, and design primers for about 50% more loci than we’d be aiming to use in the end; e.g.&nbsp;27 loci if aiming for 18. We charge $300 per locus to design and test primers, so that puts us about $8000 down the road before we find out if we ended up with nice markers. We would use about 30 samples for marker testing, so there would be no subsequent need to genotype those 30 samples, but it does mean that the first round of work would be around $5500 more expensive than a routine project on a species that we know.</p>
<p>In short, if it’s a project that might extend for a number of years, and run to many hundreds of samples, the development cost isn’t too out of line (we’ve done similar things for a multiyear gyrfalcon project that I’m quite pleased with), but for a one-off project it might strike you as too expensive.</p>
<p>With all the primer design and testing we’d need a solid 8 months to do a project like this.</p>
<p>Thoughts?</p>
<p>David</p>
</div>
<div id="tabset-1-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-2-tab">
<p>Contact: Rachel Crowhurst, Rachel.Crowhurst@oregonstate.edu</p>
<p>Hi Shannon,</p>
<p>Thanks for your interest in our lab, and apologies for the delay in responding to you.</p>
<p>I looked through the first paper that you sent (Faircloth et al.&nbsp;2006) and there are plenty of highly variable markers for bluebirds, which is great.&nbsp; However, it looks like in that paper they ran every marker individually because they were characterizing new loci.&nbsp; For us to try to estimate how much this would cost we’d need to see how many markers can be multiplexed in a single PCR reaction, as that can greatly influence the cost.&nbsp; I could try to guess based on allele sizes, but it would be most helpful to see how someone else used these markers in a previously published study.&nbsp; Unfortunately, I was unable to access the Fowler Neal thesis without a University of Arkansas login.&nbsp; Would you be able to download the thesis and send it to me as a .pdf?&nbsp; &nbsp;We typically charge about $100 per sample for species that we’ve worked on before, for up to 3 panels of markers.&nbsp; However, there is an initial cost for the labelled primers and since we do not have those on-hand, that would add approximately $4000.&nbsp; We would probably also need to build in a buffer for optimization runs, since it often takes a bit of tweaking before we can get a new primer/species system to work, even if we’re working off of a previously published study.&nbsp;</p>
<p>Full disclosure – we specialize in mammalian samples (feces, blood, tissue) and although I’ve done some sexing work on blood from seabirds I do not have experience working with feathers.&nbsp; In my reading I’ve seen published protocols wherein people use the same DNA extraction kit for feathers as we use for muscle (Qiagen DNEasy blood and tissue), with a few modifications.&nbsp; However the Faircloth paper used DNA from blood and the PCR product sizes are very large (mostly 250-350bp) and it’s unclear to me whether the feather DNA would work as well for such large amplicons.&nbsp; I’d like to see what methods and sample type were used in Jessica Fowler Neal’s study.&nbsp; Once we have a better idea of how many markers we could fit into a single PCR reaction and what extraction methods would be best to try, we would be better positioned to estimate the cost and turnaround time.</p>
<p>Best, Rachel Crowhurst</p>
</div>
<div id="tabset-1-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-3-tab">
<p>Contact: Nicole Chinnici, nchinnici@esu.edu</p>
<p>Hi Shannon</p>
<p>Thank you for your request. This is a service that we can assist&nbsp;you with. To better understand your project and get you an accurate quote, we need some additional&nbsp;information.</p>
<ol type="1">
<li><p>Will you be selecting the microsatellites for your project?</p></li>
<li><p>Will you purchase those microsatellites yourself, or would you like that included in our service?</p></li>
<li><p>What is the timeframe you are looking to complete this project?</p></li>
</ol>
<p>Thank you, Nicole</p>
</div>
<div id="tabset-1-4" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-4-tab">
<p>Contact: Barbara Fransway, bbf@arizona.edu</p>
<p>Shannon- thank you for getting back to our lab.</p>
<p>I don’t have experience extracting DNA from feathers, this would have to be piloted with some samples and then we could discuss how to proceed after DNA QC and yield assessment.</p>
<p>For the updated costs-</p>
<p>Assume- 5 multiplexes per sample- &nbsp;this will help keep the primer costs low and reduce the optimization effort. In addition, if there are markers that are “must have” vs “not critical” this will also be informative.</p>
<p>In short, rough costs are:</p>
<p>Once optimized- DNA extraction +QC ~$12.50/sample PCR $3.60/reaction (<em>5= $18) Fragment analysis $2.10/well (</em>5=$10.50)</p>
<p>So production test costs would be about $40/sample- this is Extraction, QC, PCR and Frag if we do 5 reactions per sample</p>
<p>Primer costs are ~$85-$100/locus *18= $1550-1800</p>
<p>Optimization: PCR- $3.60/reaction, gel electrophoresis estimated $60-90, and fragment analysis $2.10/sample- the quantities of PCR and Frag depend on how complex the multiplex assays are. Assume $500-600 for optimization</p>
<p>Estimated total- Pilot extraction/QC &nbsp;(n=8) $100 if successful then we proceed. DNA extraction/QC, PCR and FRAG (n=100) $4000 Data Finishing $420 Optimization $600 Primers $1800</p>
<p>So it looks like we are at about $6900 for this project. This again greatly depends on the loci selected and how we multiplex them together. In addition the pilot DNA extraction and following QC will be a critical point prior to proceeding with the samples.</p>
<p>If you would like a more official quote please register for iLab and I can create one in this system for you. See registration instructions attached.</p>
<p>Thank you and I look forward to being in contact soon. Barbara</p>
</div>
</div>
</div>
</section>
<section id="microsatellite-selection" class="level3">
<h3 class="anchored" data-anchor-id="microsatellite-selection">Microsatellite Selection</h3>
<div class="callout callout-style-simple callout-none no-icon" style="background:lightsteelblue">
<div class="callout-body d-flex">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-body-container">
<p>Characteristics for primer sets should be highly polymorphic (many alleles) to distinguish between closely related individuals, and should maintain HWE for reliable allele frequency data (i.e.&nbsp;H<sub>o</sub> ≈ H<sub>E</sub>)</p>
</div>
</div>
</div>
<section id="main-texts" class="level4">
<h4 class="anchored" data-anchor-id="main-texts">Main texts</h4>
<ul>
<li><p><strong>Gutiérrez et al.&nbsp;2022:</strong></p>
<p><a href="https://www.biorxiv.org/content/10.1101/2022.05.26.493631v1.full.pdf" class="uri">https://www.biorxiv.org/content/10.1101/2022.05.26.493631v1.full.pdf</a></p>
<ul>
<li>“Five fluorescently labeled primers were amplified: Smex9 (Ferree et al.&nbsp;2008), Smex14 (Ferree et al.&nbsp;2008), Pdo5 (Griffith et al.&nbsp;1999), Sialia36 (Faircloth et al.&nbsp;2006), and Sialia37 (Faircloth et al.&nbsp;2006)… Each primer was selected based on previous amplification success using eastern bluebird DNA (Faircloth et al.&nbsp;2006, Ferree et al.&nbsp;2008, and Stewart et al.&nbsp;2009).”</li>
</ul></li>
<li><p><strong>Stewart et al.&nbsp;2009</strong>:</p>
<p><a href="https://doi.org/10.1007/s00265-009-0862-y" class="uri">https://doi.org/10.1007/s00265-009-0862-y</a></p>
<ul>
<li>Uses Pdo5 (Griffith et al.&nbsp;1999), a loci for paternity in HOSP, as well as other loci developed for other species. Only EABL specific loci from Balenger et al 2009.</li>
</ul></li>
<li><p><strong>Balenger et al.&nbsp;2009:</strong></p>
<p><a href="https://doi.org/10.1111/j.1600-048X.2009.04521.x" class="uri">https://doi.org/10.1111/j.1600-048X.2009.04521.x</a></p>
<ul>
<li>uses Sailia37 from Faircloth 2006, and 2 new primers (Eabl007, Eabl129).</li>
</ul></li>
<li><p><strong>Ferree et al 2008</strong>:</p>
<p><a href="https://doi.org/10.1111/j.1755-0998.2008.02265.x" class="uri">https://doi.org/10.1111/j.1755-0998.2008.02265.x</a></p>
<ul>
<li>Though primers labelled as Smex, tested on NC population of S. sialis as well</li>
<li>The two primers chosen by Gutierrez 2022 had a significant deviation from Hardy–Weinberg equilibrium in this original paper— my understanding is this is not desirable?? Possibly did some trouble shooting for their population to come to this selection?</li>
</ul></li>
<li><p><strong>Faircloth et al.&nbsp;2006</strong>:</p>
<p><a href="https://doi.org/10.1111/j.1471-8286.2006.01408.x" class="uri">https://doi.org/10.1111/j.1471-8286.2006.01408.x</a></p>
<ul>
<li><u> Neal 2014</u>: chose Sialia 6, 15, 20, 23, 27, 28, 34, 36, 38 “based on successful amplification and a satisfactory number of alleles.”</li>
</ul></li>
</ul>
</section>
<section id="additional-texts" class="level4">
<h4 class="anchored" data-anchor-id="additional-texts">Additional texts</h4>
<ul>
<li><p><strong>Joly 2024:</strong> Mountain Bluebirds study, Chose Sailia 37, EABL 129, and Smex 6,8,10,13</p></li>
<li><p><strong>Dickinson et al.&nbsp;2016 &amp; Ferree et al.&nbsp;2010:</strong> selected 11 microsatellites from Ferree 2008 paper- Smex 2,4,5, 7-14</p>
<ul>
<li>In other words, they selected primers that did not deviate from HWE for western bluebirds</li>
<li>For EABL, this would be Smex 2-8,10,12,13 <em>(15 borderline HWE deviation)</em>
<ul>
<li>whitling it down more -3, 5, 6, 7, 12, 13 are best for HWE and allele no.</li>
</ul></li>
</ul></li>
<li><p><strong>Avery et al.&nbsp;2013:</strong> selected 12 loci from Faircloth 2006 (those without sig. HWE deviation and with adequate # of alleles)</p></li>
<li><p><strong>Liu 2009:</strong> Chose Sailia 11,22,27,36,37, eabl129, and MOBL47,49,53,86b from Balenger et al.&nbsp;2009 (likely an early copy? Only 49 &amp; 87 published). MOBL53 &amp; Sailia22 had HWE deviation.</p></li>
</ul>
<div class="callout callout-style-simple callout-none no-icon" style="background:thistle">
<div class="callout-body d-flex">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-body-container">
<p>Based on HWE and allele #, it seems like these are good candidates?</p>
<p>Faircloth- Sailia 11, 22, 28, 37 (trinucleotide)</p>
<p>Balenger - Eabl007, Eabl129, (Maybe MOBL49, MOBL87?) (dinucleotide)</p>
<p>Ferree - Smex 3,5,6,7,12,13 (dinucleotide)</p>
</div>
</div>
</div>
</section>
<section id="other-relevant-info" class="level4">
<h4 class="anchored" data-anchor-id="other-relevant-info">Other relevant info</h4>
<p>Eastern Bluebird whole genome assembly (LSU):</p>
<ul>
<li><a href="https://www.ncbi.nlm.nih.gov/datasets/genome/GCA_009812075.1/" class="uri">https://www.ncbi.nlm.nih.gov/datasets/genome/GCA_009812075.1/</a></li>
</ul>
<p>WGS on 6 individuals from Schroeder et al.&nbsp;2024 (Rolland Lab):</p>
<ul>
<li><p><a href="https://doi.org/10.1002/ece3.10851" class="uri">https://doi.org/10.1002/ece3.10851</a></p></li>
<li><p><a href="https://www.ncbi.nlm.nih.gov/bioproject/1040132" class="uri">https://www.ncbi.nlm.nih.gov/bioproject/1040132</a></p></li>
</ul>
<p>Genbank microsatellite sequences:</p>
<ul>
<li><a href="https://www.ncbi.nlm.nih.gov/nuccore/?term=sialia+microsatellite" class="uri">https://www.ncbi.nlm.nih.gov/nuccore/?term=sialia+microsatellite</a></li>
</ul>
</section>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>